Tests/Invoke-PxGet.Tests.ps1

```diff
@@ -127,16 +146,35 @@ Describe 'Invoke-Pxget' {
         ThenInstalled -ModuleName 'Carbon' -Version '2.10.0'
     }
 
-    It 'should pass when given an install path' {
-        New-Item -Path $(Join-Path -Path $testRoot -ChildPath 'TestPath') -ItemType 'Directory'
-        $testPath = (Join-Path -Path $testRoot -ChildPath 'TestPath').replace('\', '\\')
+    It 'should pass when given an install path that exists' {
+        $testPath = Join-Path -Path $testRoot -ChildPath 'TestPath'
+        New-Item -Path $testPath -ItemType 'Directory'
+        $contents = @"
+        {
+            "PSModules": [
+                {
+                    "Name": "NoOp",
+                    "Version": "1.0.0",
+                    "Path": $($testPath | ConvertTo-Json)
+                }
+            ]
+        }
+"@
+        GivenPxGetFile -Contents $contents
+        WhenInvokingPxGet
+        ThenSucceeded
+        ThenInstalled -ModuleName 'NoOp' -Version '1.0.0' -InstallPath $testPath
+    }
+
+    It 'should pass when given an install path that does not exist' {
+        $testPath = Join-Path -Path $testRoot -ChildPath 'NewPath'

```

Comment:
If you add another non-existent path, creating the directory will fail because the parent directory of the new item doesn't exist. You need to use the `New-Item` cmdlet's `-Force` parameter to create *all* the parent directories in a path if needed. Change this line to two lines and this test should fail:

```powershell
$testPath = Join-Path -Path $testRoot -ChildPath 'NewPath'
$testPath = Join-Path -Path $testPath -ChildPath 'AnotherPath'
```